test/integration/client_test.ts

```diff
@@ -763,4 +765,39 @@ describe.sequential('Client', function () {
       await cli.deactivate();
     }
   });
+
+  it('Should retry on network failure and eventually succeed', async ({
```

Comment:
@hackerwins 
Would you please explain why this tc doesn't assert the condition of WatchLoop?